package.json

```diff
@@ -37,7 +37,7 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "./node_modules/.bin/mocha --timeout 5000 -R spec",
+    "test": "./node_modules/.bin/mocha --timeout 9999 -R spec",
```

Comment:
Some tests take more time than another .. so if increasing the timeout fixes the issue, then I think we should increase it. If it is only one or two tests that need the extra timeout, then maybe we can define the extra timeout for those specific tests. If they are too many, then I am fine with increasing the timeout